### DISCUSSION TITLE
Difference in MTEX implementation of map_into_symmetry_reduced_zone()

### DISCUSSION THREAD

**Discussion by aditya-shukl** [pyxem/orix#454] [Q&A]
Hi . I have a set for Euler angles for rotation. e=[1.38701239 ,2.27912029, 2.67505697] in the Bunge convnention. I wanted to find the lowest angle possible after using the cubic symmetry of my crystal for this rotation. When I use the function map_into_symmetry_reduced_zone(), it gives me the axis of rotation as [ 0.8873 -0.0097  0.4611]. Whereas in MTEX on using project2FundamentalRegion, I get the axes as [-0.461087    0.887301 -0.00974407]. The result of the MTEX axes as  is correct as other orientations I have for the grain. Can you help me with I am getting two different results when I project my rotation to the fundamental zone of the cubic symmetry.  The angle of rotation is the same in both cases.

Are these two representing the same rotation somehow.  Can you help me understand. I tried doing inv(o1)*o2 and it does not give me a zero.

Let me know if my question is unclear.

Thanks
Aditya

**Comment by hakonanes** [comment]
Hi @aditya-shukl,

it turns out that the two cubic crystal *directions* are symmetrically equivalent, but not the orientations. With the latest orix release v0.11.1:

```python
>>> from orix.crystal_map import Phase
>>> from orix.quaternion import Orientation, symmetry
>>> from orix.vector import Miller
>>> o = Orientation.from_euler([1.38701239, 2.27912029, 2.67505697], symmetry=symmetry.Oh)
>>> o2 = o.map_into_symmetry_reduced_zone()
>>> o2.angle
array([0.82540229])
>>> o2.axis
Vector3d (1,)
[[ 0.8873 -0.0097  0.4611]]
>>> u_orix = Miller(xyz=o2.axis.data, phase=Phase(point_group=o2.symmetry))
>>> u_mtex = Miller(xyz=[-0.461087, 0.887301, -0.00974407], phase=u_orix.phase)
>>> u_orix.angle_with(u_mtex, use_symmetry=True)
array([0.])
>>> o_mtex = Orientation.from_axes_angles([-0.461087, 0.887301, -0.00974407], o2.angle, symmetry=symmetry.Oh)
>>> o2.angle_with(o_mtex, degrees=True)
array([39.16537806])
```

Can you give the MTEX quaternion and rotation angle of your reduced orientation, so that I can confirm that `o_mtex` is defined correctly?

I believe the different orientations returned from orix and MTEX within their respective fundamental zones is because the zones are defined differently. We're working towards sorting out any unexpected results from reduction to these zones, see e.g. https://github.com/pyxem/orix/pull/442. Note that these things take time...

**Reply by aditya-shukl** [reply]
Hello @hakonanes 

this is the ori data after applying the function
I am not sure how to get quaternions from my mtex orientation object. doing o.a ,o.b ... gives me these four numbers.
angle-47.2921
(a,b,c,d)= 0.1857, -0.7265,0.5454,-0.3745
euler=ans = orientation (m-3m → xyz)
 
  Bunge Euler angles in degree
     phi1     Phi    phi2
  117.214 47.2897 242.297
 
My initial goal for converting them to fundamental zones was to get misorientations between a set a orientations. I followd the clustering orientation tutorial which uses this function of mapping to fundamental zones. I noticed the orientations which are symmetricaly equivalent were clustered differently. Hence I raised this discussion. Thanks for your help and the nice package. :)

Best
Adi

**Reply by aditya-shukl** [reply]
To add on , the second orientaiton with which this should be symmetrically equaivalent is .

e=[2.04495978 0.82478015 4.23017009].

I am trying to find a way to prove this using orix.
